setup.py

```diff
@@ -42,7 +42,8 @@
     extras_require={
         "test": [
             "plone.app.testing",
-            "plone.app.contenttypes"
+            "plone.app.contenttypes",
+            "plone.app.robotframework[debug]",
```

Comment:
There are no robot tests, so this should not be added.

Reply:
@mauritsvanrees if i remove the `"plone.app.robotframework[debug]",` then the tests are broken.

```
> bin/test

Module: plone.portlet.collection.tests.test_functional_doctest

Traceback (most recent call last):
  File "/opt/development/collective/plone.portlet.collection/plone/portlet/collection/tests/test_functional_doctest.py", line 6, in <module>
    from plone.portlet.collection.testing import PLONE_PORTLET_COLLECTION_FUNCTIONAL_TESTING
  File "/opt/development/collective/plone.portlet.collection/plone/portlet/collection/testing.py", line 2, in <module>
    from plone.app.contenttypes.testing import PLONE_APP_CONTENTTYPES_FIXTURE
  File "/home/1letter/.buildout/eggs/cp38/plone.app.contenttypes-3.0.0a11-py3.8.egg/plone/app/contenttypes/testing.py", line 4, in <module>
    from plone.app.event.testing import PAEvent_FIXTURE
  File "/home/1letter/.buildout/eggs/cp38/plone.app.event-4.0.0a8-py3.8.egg/plone/app/event/testing.py", line 3, in <module>
    from plone.app.robotframework.testing import AUTOLOGIN_LIBRARY_FIXTURE
ModuleNotFoundError: No module named 'plone.app.robotframework'


Module: plone.portlet.collection.tests.test_portlet_collection

Traceback (most recent call last):
  File "/opt/development/collective/plone.portlet.collection/plone/portlet/collection/tests/test_portlet_collection.py", line 6, in <module>
    from plone.portlet.collection.testing import (
  File "/opt/development/collective/plone.portlet.collection/plone/portlet/collection/testing.py", line 2, in <module>
    from plone.app.contenttypes.testing import PLONE_APP_CONTENTTYPES_FIXTURE
  File "/home/1letter/.buildout/eggs/cp38/plone.app.contenttypes-3.0.0a11-py3.8.egg/plone/app/contenttypes/testing.py", line 4, in <module>
    from plone.app.event.testing import PAEvent_FIXTURE
  File "/home/1letter/.buildout/eggs/cp38/plone.app.event-4.0.0a8-py3.8.egg/plone/app/event/testing.py", line 3, in <module>
    from plone.app.robotframework.testing import AUTOLOGIN_LIBRARY_FIXTURE
ModuleNotFoundError: No module named 'plone.app.robotframework'
```

Reply:
Ah, that one again.
Two possible solutions.

- Add `plone.app.contenttypes[test]` to the test requirements. That pulls in the needed extra packages, including p.a.robotframework. Not ideal, but looks slightly nicer than explicitly including it ourselves.
- Better: do not use `PLONE_APP_CONTENTTYPES_FIXTURE` in `testing.py`, but use `from plone.app.testing import PLONE_FIXTURE`.

Reply:
@mauritsvanrees i have decided me for option two. Test are running.